### PR TITLE
Settings: MEDIA ROOT added for storing media files

### PR DESCRIPTION
**Pull request by deshraj**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -82,3 +82,7 @@ bower_components/
 
 # Challenge data files
 challenge_data/*
+
+# Media Files
+media/
```
Comment (TroJan) [inline .gitignore]: can you explain why is this directory ignored ?

File: settings/dev.sample.py [hunk 2]
```diff
@@ -39,3 +39,5 @@
         'BACKEND': 'django.core.cache.backends.dummy.DummyCache',
     }
 }
+
+MEDIA_URL = "/media/"
```
Comment (TroJan) [inline settings/dev.sample.py]: what about other settings?

File: .gitignore (hunk 1 above)
Comment (deshraj) [inline .gitignore]: For development, all the media files will go in this directory and hence we don't want anyone to add the media directory to staging area.

File: settings/dev.sample.py (hunk 2 above)
Comment (deshraj) [inline settings/dev.sample.py]: I don't think we need any other settings. Can you please elaborate a bit more?

**Pull request merged**